docker_deploy.sh

```diff
@@ -2,7 +2,7 @@
 
 source environment/production.env
 
-docker run -p 8000:8000 \
+docker run -p 80:80 \
```

Comment:
just wondering, why was it changed to 80:80?

Reply:
I believe because the Dockerfile launch command was changed to: `CMD ["uvicorn", "pephub.main:app", "--host", "0.0.0.0", "--port", "80"]` so I wanted the ports to match.